### PR TITLE
Feat/national headline stats (#105)

### LINKED ISSUE #105: National Headline stats
Add two headline figures: 
- 1. Last output from pv live, The latest black dashed value
- 2. Forecast value for the same time, the last solid yellow line

Already in UI:
- 3. The “Forecast next” value is the same as the current value we are displaying in the UI.

The times (10.30) in the descriptions are always the same as each other and change through the day. (Same as white line)

When a user moves the time (selected_time):
- 1. the PV live value at that time, if the time is in the future, just display the latest PV value. If both 'PV live initial' and 'updated' are present, select the 'updated' value
- 2. the OCF Forecast value at that time
- 3. Always the next OCF forecast value after 'now';

Screenshot 2022-06-16 at 12 21 11

### PR DESCRIPTION
**Pull request by dlinah**
Fixes #105

### CODE REVIEW

**Comment by peterdudfield** [comment]
Screenshot 2022-06-28 at 10 55 34

Seems to be a few issues with the stats. 
1. Forecast next is 0
2. actual seems to be at 10.00, but it should be for 10.30

Soryy @dlinah if you havent finished this feature yet

File: apps/nowcasting-app/components/charts/forecast-header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+const ForecastHeader: React.FC<ForecastHeaderProps> = ({
+  pvUpdatedData,
+  pvLiveData,
+  pvForecastData,
+  selectedTime,
+}) => {
+  const timeNow = useTimeNow();
+  const nextPvForecastInGW = MWtoGW(
+    pvForecastData?.find((fc) => formatISODateString(fc.targetTime) === timeNow)
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/charts/forecast-header/index.tsx:24]: this value return undefined

**Review by Alaatohamy — CHANGES_REQUESTED** [review]

**Comment by peterdudfield** [comment]
Ive now fixed it to what I think it should be
default screnn

Screenshot 2022-06-28 at 21 10 09

select time, before now

Screenshot 2022-06-28 at 21 10 18

selected time after no
Screenshot 2022-06-28 at 21 10 26

**Comment by peterdudfield** [comment]
Sorry @dlinah I took ages tryin to fix lint there. Ill hand this one back over to you if thats ok. Let me know if I have done anything you dont understand. 

Sorry I should have gone onto another branch and given you a PR

**Comment by peterdudfield** [comment]
I decided to push this onto Development, so users can see it. 
Would @Alaatohamy or @dlinah be able to review my code (and correct it if needed)

Thanks

File: apps/nowcasting-app/components/hooks/use-time-now.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "react";
+import useGlobalState, { get30MinNow, getFloor30MinNow } from "../globalState";
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/hooks/use-time-now.tsx:2]: remove getFloor30MinNow it's not used anymore here

File: apps/nowcasting-app/components/globalState.tsx [hunk 3]
```diff
@@ -12,14 +13,29 @@ export function get30MinNow() {
   }
   return date.toISOString();
 }
+
+export function getFloor30MinNow() {
```
Comment (Alaatohamy) [inline apps/nowcasting-app/components/globalState.tsx:17]: you are not using this hook, can we delete it if it's not necessary please

**Comment by Alaatohamy** [comment]
@peterdudfield thank you so much, good job

File: apps/nowcasting-app/components/hooks/use-time-now.tsx (hunk 2 above)
Comment (peterdudfield) [reply apps/nowcasting-app/components/hooks/use-time-now.tsx:2]: Thanks

File: apps/nowcasting-app/components/globalState.tsx (hunk 3 above)
Comment (peterdudfield) [reply apps/nowcasting-app/components/globalState.tsx:17]: thanks

**Comment by peterdudfield** [comment]
ive put PR comments chnages here- https://github.com/openclimatefix/nowcasting/pull/139

**Pull request merged**